include/highfive/H5PropertyList.hpp

```diff
@@ -498,6 +498,32 @@ class UseCollectiveIO {
 };
 #endif
 
+enum CreationOrder {
+    Tracked = H5P_CRT_ORDER_TRACKED,
+    Indexed = H5P_CRT_ORDER_INDEXED,
+};
```

Comment:
Do we encapsulate other values like this? I didn't do this for the version bounds, wondering if this will incur more maintenance later?

At a minimum, seems like the `enum` should be inside `LinkCreationOrder`?

Reply:
Yes, for example: https://github.com/BlueBrain/HighFive/blob/master/include/highfive/bits/H5DataType_misc.hpp#L36-L38

let's discuss ;)

I think, this is more convenient for user (to hide HDF5 ^^)

Reply:
Should we encapsulate more? Like the version bounds?

Anyways, I'd be fine as long as it's contained somewhere in a class, so we don't have a free `Tracked` without context :D

Reply:
I agree with the class. I just wait to know if I should delete it, before moving it.